lib/api/core/plugins/pluginsManager.js

```diff
@@ -418,7 +421,7 @@ class PluginsManager {
         }, timeoutDelay);
       }
 
-      const cb = (err, pipeResponse) => {
+      const cb = (err, pipeResponse, ...info2) => {
```

Comment:
You should find a better name than `info2`

Reply:
Like what